media/css/firefox/channel.less

```diff
@@ -184,17 +184,19 @@
 // Platform navigation
 
 .platform-nav {
+    .sticky;
```

Comment:
I think rather than apply this everywhere we should instead use a feature detect using `@supports`:

```
@supports (position: sticky) {
    .platform-nav {
        .sticky;
    }
}
```

This way we can also apply the `:target` top padding for just the browsers that support the sticky header.